src/main/java/io/debezium/connector/vitess/SourceInfo.java

```diff
@@ -41,6 +41,11 @@ protected Instant timestamp() {
 
     @Override
     protected String database() {
+        // Override DATABASE_NAME_KEY to empty string and in favor of KEYSPACE_NAME_KEY
+        return "";
```

Comment:
nit: `return null;`
I saw other connectors do that in some cases and AbstractSourceInfoStructMaker will convert it to "".

Reply:
Sure, done.

it took me a second to remember String can be `null` in Java whereas string is not nil :)